src/game/inject.c

```diff
@@ -1484,6 +1500,30 @@ static void Inject_RoomDoorEdits(INJECTION *injection)
     }
 }
 
+static void Inject_ItemRotations(INJECTION *injection)
+{
+    if (injection->version < INJ_VERSION_4) {
+        return;
+    }
+
+    INJECTION_INFO *inj_info = injection->info;
+    MYFILE *fp = injection->fp;
+
+    int16_t item_num;
+    int16_t y_rot;
+    for (int i = 0; i < inj_info->item_rotation_count; i++) {
+        File_Read(&item_num, sizeof(int16_t), 1, fp);
+        File_Read(&y_rot, sizeof(int16_t), 1, fp);
+
+        if (item_num < 0 || item_num >= g_LevelItemCount) {
+            LOG_WARNING("Item number %d is outwith level item range", item_num);
```

Comment:
outwith? :D

Reply:
Lol Lahm helped out on this. Sorry to change native tongue, but I'll change it to make it easier to understand. 😢 😆